### PR TITLE
fix e2e test for tables in chapter 8, 9, and 10

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

File: chapter08/e2e/tables/element.e2e-spec.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {browser, by, element, ElementFinder, ElementHelper, ProtractorBy} from 'protractor';
 import {By as webdriverBy, WebElement} from 'selenium-webdriver';
 
-describe('element methods', () => {
+fdescribe('element methods', () => {
```
Comment (jesselpalmer) [inline chapter08/e2e/tables/element.e2e-spec.ts:4]: @cnishina was `fdescribe` suppose to be left in there?

File: chapter10/test_debugging/e2e/contacts.e2e-spec.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ describe('contacts test', () => {
 
   fit('should open the dialog with waitForAngular', async () => {
```
Comment (jesselpalmer) [inline chapter10/test_debugging/e2e/contacts.e2e-spec.ts:62]: @cnishina how about `fit`?

Comment (heathkit) [reply chapter10/test_debugging/e2e/contacts.e2e-spec.ts:62]: whoops, that's my fault.

Comment (cnishina) [reply chapter10/test_debugging/e2e/contacts.e2e-spec.ts:62]: It should be fixed. The `fit` should be `it`. I'll get to it tonight.

File: chapter08/e2e/tables/element.e2e-spec.ts (hunk 1 above)
Comment (cnishina) [reply chapter08/e2e/tables/element.e2e-spec.ts:4]: Oh that's me. Going to fix this too.

**Pull request merged**